docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:14.04
```

Comment:
This image is redundant with [jenkinsciinfra/packaging](https://github.com/jenkins-infra/docker-packaging/blob/master/Dockerfile#L4), while I admit I am the one who split the definition into a different git repository, I think it would make sense to either remove this Dockerfile from here or transfer the work being done on jenkins-infra/docker-packaging here. Personally I would be in favor of keeping jenkins-infra/docker-packaging as there is already a process to build and publish this image and just reuse jenkinsciinfra/packaging and also this image is used in the release environment